api_server.py

```
@@ -74,38 +90,55 @@ def format_hours(minutes: Optional[int]) -> Optional[str]:
+    return {
+        "available": True,
+        "duration": format_hours(duration_min),
+        "end_time_display": display,
+    }
+
+
 def get_availability(entity_id: int, table: str, now: datetime) -> Dict:
     col = "crew_id" if table == "crew_availability" else "appliance_id"
     with get_db() as conn:
         curr = conn.execute(
             f"SELECT end_time FROM {table} WHERE {col} = ? AND start_time <= ? AND end_time > ? LIMIT 1",
```

Ruff (0.15.1)

[error] Possible SQL injection vector through string-based query construction

(S608)